setup.py

```diff
@@ -258,7 +257,10 @@ def finalize_options(self):
             include_dirs=[model_dir]),
         Extension('gensim.models.fasttext_inner',
             sources=['./gensim/models/fasttext_inner.c'],
-            include_dirs=[model_dir])
+            include_dirs=[model_dir]),
+        Extension('gensim.models.ldamodel_inner',
+            sources=['./gensim/models/ldamodel_inner.c'],
```

Comment:
Maybe better name for it is `_matutils` (because this isn't model at all, this only concrete math functions)?

Reply:
Also, move this file on level of `matutils.py` ? wdyt?